# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/scroll-animations/css/animation-timeline-view-functional-notation.tentative.html.ini
# Commit: 425cbe3c033b
# Full Hash: 425cbe3c033bfc25795979513ebd2bb9fda7338d
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2024-11-28 21:54:11
# Regressor Bug: 1933829
# File Overlap Count: 1
# Description:
#   Bug 1933829 - [wpt-sync] Update web-platform-tests to 2d5d222739fdf91743f8cbc58094976a613194e7, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 2d5d222739fdf91743f8cbc58094976a613194e7
# ==============================================================================

diff -r 732bcd7ac971 -r 425cbe3c033b testing/web-platform/meta/scroll-animations/css/animation-timeline-view-functional-notation.tentative.html.ini
--- a/testing/web-platform/meta/scroll-animations/css/animation-timeline-view-functional-notation.tentative.html.ini	Wed Nov 27 22:05:28 2024 +0000
+++ b/testing/web-platform/meta/scroll-animations/css/animation-timeline-view-functional-notation.tentative.html.ini	Wed Nov 27 22:05:56 2024 +0000
@@ -1,67 +1,53 @@
 [animation-timeline-view-functional-notation.tentative.html]
   expected:
-    if os == "mac": [OK, TIMEOUT]
     if os == "linux": [TIMEOUT, OK]
   [animation-timeline: view()]
     bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1824875
     expected:
-      if os == "mac": [FAIL, TIMEOUT, NOTRUN]
       if os == "linux": [FAIL, TIMEOUT]
       FAIL
 
   [animation-timeline: view(50px)]
     bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1824875
     expected:
-      if os == "mac": [FAIL, TIMEOUT, NOTRUN]
-      if os == "linux": [TIMEOUT, FAIL, NOTRUN]
+      if os == "linux": [FAIL, TIMEOUT, NOTRUN]
       FAIL
 
   [animation-timeline: view(auto 50px)]
     bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1824875
     expected:
-      if os == "mac": [FAIL, TIMEOUT, NOTRUN]
-      if os == "linux": [NOTRUN, FAIL, TIMEOUT]
+      if os == "linux": [FAIL, TIMEOUT, NOTRUN]
       FAIL
 
   [animation-timeline: view(inline)]
     bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1824875
     expected:
-      if os == "mac": [FAIL, NOTRUN]
-      if os == "linux": [NOTRUN, FAIL, TIMEOUT]
+      if os == "linux": [FAIL, TIMEOUT, NOTRUN]
       FAIL
 
   [animation-timeline: view(), view(inline)]
     bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1824875
     expected:
-      if os == "mac": [FAIL, NOTRUN]
       if os == "linux": [NOTRUN, FAIL]
       FAIL
 
   [animation-timeline: view(inline) changes to view(inline 50px)]
     bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1824875
     expected:
-      if os == "mac": [FAIL, NOTRUN]
       if os == "linux": [NOTRUN, FAIL, TIMEOUT]
       FAIL
 
   [animation-timeline: view(x)]
     expected:
-      if os == "mac": [FAIL, NOTRUN]
-      if os == "linux": [NOTRUN, FAIL, TIMEOUT]
+      if os == "linux": [FAIL, TIMEOUT, NOTRUN]
       FAIL
 
   [animation-timeline: view(y)]
     expected:
-      if os == "mac": [FAIL, NOTRUN]
-      if os == "linux": [NOTRUN, FAIL]
+      if os == "linux": [FAIL, NOTRUN]
       FAIL
 
   [animation-timeline: view(x 50px)]
     expected:
-      if os == "mac": [FAIL, NOTRUN]
-      if os == "linux": [NOTRUN, FAIL]
+      if os == "linux": [TIMEOUT, FAIL, NOTRUN]
       FAIL
-
-  [animation-timeline: view(inline) changes to view(inline 50px), withouttimeline range name]
-    expected:
-      if os == "mac": [PASS, TIMEOUT]